webapp-django/crashstats/settings/base.py

```diff
@@ -102,6 +102,7 @@ def path(*dirs):
 
 
 MIDDLEWARE_CLASSES = (
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
Do we know if this change affects our current server environments?

Reply:
Not with confidence other than theory. 
In Nginx we have it so that every /static URL gets taken care of by Nginx as a straight file serving. 
https://github.com/mozilla/socorro-infra/blob/master/puppet/modules/socorro/templates/etc_nginx/conf_d/socorro-webapp.conf.erb#L49-L54
Whitenoise doesn't do it by prefix. It basically looks at the `request.path_info` and tries to match that whole path to a file on disk (based on that STATIC_ROOT)